config/routes.rb

```diff
@@ -6,7 +6,7 @@
 
   resources :categories, only: [:show, :index]
   resources :items
-  get "cart_items/create"
+  get 'cart_items/create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.